README.md

```diff
@@ -15,21 +14,21 @@ To make it easier to contribute to the content, Linaro provides a couple of Dock
 
 To build the site:
 
-```
+```bash
```

Comment:
I'd actually prefer **not** to have syntax highlighting in these sections because the different colours, I think, make it harder to see than using just plain black.